lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -253,6 +258,21 @@ class ReportUIFeatures {
     }
   }
 
+  /**
+   * Keyup handler for the document.
+   * @param {KeyboardEvent} e
+   */
+  onKeyUp(e) {
+    if (e.key === 'd') {
```

Comment:
this seems fine for development, but I don't think we should leave it in.
either remove now that we're done with this PR or remove in a week?

Reply:
done